modules/apps/27-interchain-accounts/host/types/msgs_test.go

```diff
@@ -6,68 +6,47 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types"
```

Comment:
super duper nit -- group imports

Reply:
oh damn got hit with the super duper variety

Reply:
shouldn't the linter/formatter resolve this automatically?

Reply:
I want to say it doesn't separate by lines, it definitely has certain rules. Tbh haven't looked much into how the formatter for VSCode is configured for import paths.